sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/CodeCoverageAnnotation.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.phonenumbers;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import com.azure.communication.phonenumbers.CodeCoverageAnnotation.Generated;
+
+/** Class used for code coverage. */
+@Generated
+public class CodeCoverageAnnotation {
```

Comment:
How is this file generated? It says that it is code generated, which means it should be in a /generated file/